QuadradiusR/scripts/network_handler.gd

```diff
@@ -116,20 +116,21 @@ func accept_and_join_game(game_invite_id: String, cb: FuncRef):
 
 func _accept_and_join_game_1(message: Message, args: Dictionary):
 	# After accept_game_invite
-	var game_id = ""
+#	var game_id = ""
 	# FIXME: clean up this mess, HTML5 and executables
 	#   have different behaviors when it comes to redirects
-	if message.result == 200:
-		game_id = message.body['id']
-	elif message.result == 303:
-		for header in message.headers:
-			if 'location' in header:
-				game_id = header.substr(16, len(message.headers[0]))
-				break
-	join_game(game_id, args['cb'])
+	assert(message.result == 200)
```

Comment:
I don't think that we should handle it in this manner, unless it's only for testing. But if we want to actually handle the error, we need to check this manually.

The reasons for this are:
1. asserts are only executed in debug mode, not in release mode
2. you can't really do anything if it fails, it will just halt the program

For debug reasons the ```push_error``` ([doc](https://docs.godotengine.org/pl/stable/classes/class_@gdscript.html#class-gdscript-method-push-error)) might be worth considering.

Reply:
Sure, will change that